src/OpenTelemetry.Exporter.OneCollector/.publicApi/PublicAPI.Unshipped.txt

```diff
@@ -1,6 +0,0 @@
-OpenTelemetry.Exporter.OneCollector.OneCollectorLogExporterOptions.DefaultEventNamespace.get -> string!
-OpenTelemetry.Exporter.OneCollector.OneCollectorLogExporterOptions.DefaultEventNamespace.set -> void
-OpenTelemetry.Exporter.OneCollector.OneCollectorLogExporterOptions.EventFullNameMappings.get -> System.Collections.Generic.IReadOnlyDictionary<string!, string!>?
-OpenTelemetry.Exporter.OneCollector.OneCollectorLogExporterOptions.EventFullNameMappings.set -> void
```

Comment:
FYI I checked in with @flaviocdc to verify his team had consumed `1.10.0-alpha.1` and has been successfully using this API since it was released.